c2corg_ui/format/img.py

```diff
@@ -0,0 +1,89 @@
+'''
+c2corg img Extension for Python-Markdown
+==============================================
+
+Converts tags like
+[img=<id>(<options>)(/)?](<caption>)?([/img])?
+to advanced HTML img tags.
+'''
+
+from markdown.extensions import Extension
+from markdown.inlinepatterns import Pattern
+from markdown.util import etree
+
+
+class C2CImageExtension(Extension):
+
+    def __init__(self, *args, **kwargs):
+        self.config = {
+            "api_url": ['', 'Base URL of the API. Defaults to ""']
+        }
+
+        super(C2CImageExtension, self).__init__(*args, **kwargs)
+
+    def extendMarkdown(self, md, md_globals):  # noqa
+        self.md = md
+
+        IMG_RE = r'\[img=(\d+)([a-z_ ]*)(/\]|\](.*?)\[/img\])'  # noqa
+        pattern = C2CImage(IMG_RE, self.getConfigs())
+        pattern.md = md
+        # append to end of inline patterns
+        md.inlinePatterns.add('c2cimg', pattern, "<not_strong")
```

Comment:
``not_strong`` is a standard python-markdown inline patterns https://github.com/waylan/Python-Markdown/blob/master/markdown/inlinepatterns.py#L79

I have no clue what it does :P 
But this syntax ``"<not_strong"`` says that the current extension must be included/run before the ``not_strong`` pattern.
Actually I have done as in the standard wikilinks extension: https://github.com/waylan/Python-Markdown/blob/master/markdown/extensions/wikilinks.py#L51
because this argument is required and this value seems to work fine here as well.